### PR TITLE
Features/adjust logo

### PR DESCRIPTION
**Pull request by AliceGdj**
Adjust logo in navbar and homepage

### CODE REVIEW

File: src/views/home.hbs [hunk 1]
```diff
@@ -4,11 +4,9 @@
       <div class="row">
         <div class="col-sm-8">
           <h1 class="home-logo">
-            <span class="home-logo-d">Die</span>
-            <span class="home-logo-l">Lern
-              <img class="home-logo-img" src="/img/home/logo.png">
+            <span>
```
Comment (jarjan) [inline src/views/home.hbs]: Maybe we don't need this `span`

**Comment by pheeria** [comment]
[Adjust Logo](https://trello.com/c/SxNzeZzg/67-adjust-logo)

File: src/views/home.hbs (hunk 1 above)
Comment (AliceGdj) [reply src/views/home.hbs]: Ok done

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "node"
+  - "10.11.0"
```
Comment (jarjan) [inline .travis.yml]: This one specifies on which version of node travis should run project and test.

Set up this cause jest was failing with latest stable node version.

**Comment by jarjan** [comment]
@starmaerker @hikes101 please check this PR

We merged it to continue work on html/css.

**Pull request merged**